src/analysis/retail/druid/balance/modules/spells/GatheringStarstuff.tsx

```diff
@@ -0,0 +1,263 @@
+import { formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, {
+  EventType,
+  ApplyBuffEvent,
+  ApplyBuffStackEvent,
+  RemoveBuffEvent,
+  RefreshBuffEvent,
+  FightEndEvent,
+  CastEvent,
+  DamageEvent,
+} from 'parser/core/Events';
+import { currentStacks } from 'parser/shared/modules/helpers/Stacks';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { TIERS } from 'game/TIERS';
+import { calculateEffectiveDamage } from 'parser/core/EventCalculateLib';
+import { SpellLink } from 'interface';
+import ItemPercentDamageDone from 'parser/ui/ItemPercentDamageDone';
+
+const GATHERING_STARSTUFF = {
+  MAX_STACKS: 3,
+  DAMAGE_PER_STACK: 0.2,
+  STARFIRE_DELAY: 100,
+  WRATH_DELAY: 3200,
+  WRATH_MIN: 100,
+
+  AFFECTED_DAMAGE: [SPELLS.WRATH_MOONKIN, SPELLS.STARFIRE],
+  AFFECTED_CAST: [SPELLS.WRATH_MOONKIN, SPELLS.STARFIRE],
+  TRIGGER_CAST: [SPELLS.STARSURGE_MOONKIN, SPELLS.STARFALL_CAST],
+};
+
+class GatheringStarstuff extends Analyzer {
+  buffStacks: number[][];
+  lastStacks = 0;
+  lastUpdate = this.owner.fight.start_time;
+  totalDamage = 0;
+  stacksWhenCast: number[][];
+  alreadyMaxStacksWhenCast = 0;
+  gastBuffedAbilities: { [key: number]: number } = {};
```

Comment:
actually instead of putting this in a Tuple like this I would just have 2 variables to keep it clear and simple
```suggestion
  buffedStarsurgesCast: number;
  buffedStarfallsCast: number;
```